jdaviz/configs/specviz/plugins/parsers.py

```diff
@@ -81,7 +81,7 @@ def specviz_spectrum1d_parser(app, data, data_label=None, format=None, show_in_v
     if spectrum_viewer_reference_name in app.get_viewer_reference_names():
         sv = app.get_viewer(spectrum_viewer_reference_name)
         for layer_state in sv.state.layers:
-            if layer_state.visible:
+            if layer_state.visible and hasattr(sv.state, "x_display_unit"):
```

Comment:
Had to add this because line 85 kept failing for me locally.

Reply:
do you not have recent-enough glue/glue-jupyter?